src/Orchard.Cms.Web/Modules/Orchard.OpenId/Views/Admin/Create.cshtml

```diff
@@ -91,14 +158,72 @@
     </fieldset>
 </form>
 <script at="Foot" type="text/javascript">
-//<![CDATA[
+    //<![CDATA[
+    $("a > input.form-check-input").on("click", function (e) {
```

Comment:
I need this rule to avoid problems to make work collapse when an a checkbox is wrapped with an anchor. 
I use this rule in all the cshtmls within the module. Do you agree if I move it to admin.js ?